package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-core",
-  "version": "2.6.0-beta.2",
+  "version": "2.6.0-beta.3",
```

Comment:
ah, one last little 'gotcha' with these github changes I forgot to mention (sorry!)
Now that we've changed the version in `package.json`, if you run `npm i` it will update the autogenerated `package-lock.json` and update the version there as well. Could you do that really quick + add it to the PR? The change looks good otherwise!